src/runtime/lpython/lpython.py

```diff
@@ -84,6 +84,18 @@ def __init__(self, type, dims):
 Union = ctypes.Union
 Pointer = PointerType("Pointer")
 
+
+class Intent:
+    def __init__(self, type):
+        self._type = type
+
+    def __getitem__(self, params):
+        return params
+
+In = Intent("In")
+Out = Intent("Out")
+InOut = Intent("InOut")
```

Comment:
I disabled the auto-merge. I think/guess this needs a review. I would like to know if this approach is fine for supporting `In`, `Out`, `InOut` in `CPython`.